ui/components/ui/update-nickname-popover/update-nickname-popover.stories.js

```diff
@@ -1,44 +1,70 @@
-import React, { useState } from 'react';
-import { action } from '@storybook/addon-actions';
-import { text } from '@storybook/addon-knobs';
+import { useArgs } from '@storybook/client-api';
+import React from 'react';
 import Button from '../button';
+import README from './README.mdx';
 import UpdateNicknamePopover from '.';
 
 export default {
   title: 'Components/UI/UpdateNickname',
```

Comment:
I know this is in the old code but it's best if the story title is the same as the component name. Could you please update this as well 

```suggestion
  title: 'Components/UI/UpdateNicknamePopover',
```